Resources/Prototypes/Entities/Objects/Devices/forensic_scanner.yml

```diff
@@ -0,0 +1,22 @@
+- type: entity
+  name: forensic scanner
+  parent: BaseItem
+  id: ForensicScanner
+  description: A handheld device that can scan objects for fingerprints and fibers
+  components:
+  - type: Sprite
+    sprite: Objects/Devices/forensic_scanner.rsi
+    state: forensicnew
+  - type: Clothing
+    size: 5
+    sprite: Objects/Devices/forensic_scanner.rsi
+    quickEquip: false
+    Slots:
+    - Belt
+  - type: ActivatableUI
```

Comment:
Since you already have this you can delete the ActivateInWorld function I commented on earlier and it should just work(TM)